scala-native-jdbc-duckdb/src/main/resources/scala-native/duckdb.c

```diff
@@ -0,0 +1,7 @@
+#include <string.h>
+#include "duckdb.h"
+
+duckdb_data_chunk __sn_wrap_duckdb_duckdb_fetch_chunk(duckdb_result *result) {
+ return duckdb_fetch_chunk(*result);
+};
+
```

Comment:
I had some issues with some of the helpers generated by sn-bindgen, so I left only this one (which is the only one that was needed so far)

Reply:
From what I see the function just calls an externed function working on opaque structs, can be written differently so we avoid having to commit `duckdb.h`?
Alternatively, can duckdb.h be downloaded at runtime and added to generated resources?
What do you think?